assets/src/blocks/Gallery/GalleryBlock.js

```diff
@@ -44,7 +50,18 @@ export const registerGalleryBlock = () => {
       html: false, // Disable "Edit as HTMl" block option.
     },
     edit: GalleryEditor,
-    save: frontendRendered(BLOCK_NAME),
+    save: (props) => {
+      const attributes = {...props.attributes};
```

Comment:
I think this could be shortened, you could probably write `({ attributes })` instead of `(props)` in the save function declaration and then use it directly?